packages/simulators/pubspec.yaml

```diff
@@ -0,0 +1,5 @@
+name: simulators
```

Comment:
Maybe call it `ios_simulators`? As a global package name, "simulators" does not sound specific enough.

Reply:
I'll add the android emulator soon :)